### DISCUSSION TITLE
having a problem with REI

### DISCUSSION THREAD

**Discussion by Gordonramsmee** [shedaniel/RoughlyEnoughItems#1057] [Q & A] [answered]
[crash-2022-08-05_12.34.39-client.txt](https://github.com/shedaniel/RoughlyEnoughItems/files/9268175/crash-2022-08-05_12.34.39-client.txt)

Im not sure why but the mods been crashing my game as it starts up. im not sure if its a mod that I need or sum

**Comment by shedaniel** [comment, marked as answer]
You need to install Architectury,
